lib/screens/send_flow/send_amount/send_amount_bloc.dart

```diff
@@ -101,7 +101,11 @@ class SendAmountBloc extends Disposable {
     );
 
     final state = SendAmountBlocState(_fee);
-    _streamController.add(state);
+    try {
+      _streamController.add(state);
```

Comment:
Could we check `_streamController.isClosed` before adding instead?

Reply:
`tryAdd` works here too!